lingpy3/algorithm/cython/_calign.py

```diff
@@ -165,7 +165,7 @@ def globalign(
             almB += [seqB[i-1]]
             i -= 1
             j -= 1
-        else:
+        else:  # pragma: no cover
```

Comment:
just quickly checked this: reversing will cover the parts, I'll add this in my next PR in the test_script.